File: Data Structures & Algorithms/counting-bits/submission-0.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        res = [0]
        for i in range(1,n+1):
            if i % 2 == 0:
                temp = res[i//2] 
                res.append(temp)
            else:
                res.append(res[i-1] + 1)
        return(res)
        

File: Data Structures & Algorithms/max-water-container/submission-0.js
class Solution {
    /**
     * @param {number[]} heights
     * @return {number}
     */
    maxArea(heights) {
        let res = 0;
        for(let i = 0; i < heights.length-1; i++){
            for(let j = i + 1; j < heights.length; j++){
               let k = (Math.min(heights[i],heights[j])) * (j - i);
               res = Math.max(res,k);
            }
        }
        return(res)
    }
}
